### PR TITLE
Signup/login endpoint google auth

### PR DESCRIPTION
**Pull request by vinit717**
**Description**
Added /auth/google/login for google authentication

### CODE REVIEW

File: migrations/20230912194711_init.up.sql
Comment (yesyash) [inline migrations/20230912194711_init.up.sql:1]: - @iamitprakash @vinit717 is this the final schema for users table for tiny site?

File: routes/main.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	UserRoutes(v1, db)
+	AuthRoutes(v1, db)
+
+	return router
+}
+
+func Listen(listenAddress string, db *bun.DB) {
+	router := SetupV1Routes(db)
+
+	// TODO: Configure CORS properly to allow only access from certain origins
```
Comment (yesyash) [inline routes/main.go:22]: - do we want to handle this here or in the next pr?

File: migrations/20230912194711_init.up.sql
Comment (vinit717) [reply migrations/20230912194711_init.up.sql:1]: Yes as @iamitprakash has context regarding wise app so he want to do similar here

File: routes/main.go (hunk 1 above)
Comment (vinit717) [reply routes/main.go:22]: Yes 
As want not want PR to be large try to give short PR

**Review by yesyash — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**